pl_examples/domain_templates/imagenet.py

```diff
@@ -74,39 +72,21 @@ def forward(self, x):
     def training_step(self, batch, batch_idx):
         images, target = batch
         output = self(images)
-        loss_val = F.cross_entropy(output, target)
+        loss_train= F.cross_entropy(output, target)
```

Comment:
```suggestion
        loss_train = F.cross_entropy(output, target)
```
whitespace, check the code formatter for remaining issue with PEP